Superposition/Superposition.ipynb

```diff
@@ -560,10 +584,16 @@
    "name": "iqsharp"
   },
   "language_info": {
-   "file_extension": ".qs",
```

Comment:
Looks like something weird happened to the Notebook - language info should keep indicating Q#, not switch to Python. Did you open and edit it as Q# Notebook, or did you run into any trouble doing that? Continuous integration also indicates that the file is no longer a properly formatted JSON file. Could you please look into that?